recipes/fmilibrary/all/patches/2.4.1-009-link-zlib-in-fmilib.patch

```diff
@@ -0,0 +1,28 @@
+From 9478e541f41e7c9df0fb37a459c093e41848ae2e Mon Sep 17 00:00:00 2001
+From: Joakim Haugen <joakim.haugen@sintef.no>
+Date: Mon, 30 Oct 2023 10:11:24 +0100
+Subject: [PATCH] Link zlib in a test
+
+---
+ Config.cmake/fmizip.cmake | 3 ++-
+ 1 file changed, 2 insertions(+), 1 deletion(-)
+
+diff --git a/Config.cmake/fmizip.cmake b/Config.cmake/fmizip.cmake
+index 7618d51..213b5f8 100644
+--- a/Config.cmake/fmizip.cmake
++++ b/Config.cmake/fmizip.cmake
+@@ -42,9 +42,10 @@ set(FMIZIPHEADERS
+ #include_directories("${FMILIB_THIRDPARTYLIBS}/zlib/lib/VS2005/win32")
+ 
+ find_package(minizip CONFIG REQUIRED)
++find_package(ZLIB CONFIG REQUIRED)
```

Comment:
Is it a limitation from Conan zlib lib package, or the upstream is just missing it and should fix its cmake file instead?

Reply:
fmilib builds it internally and links it. Patch 005 removes this linking and patch 002 includes source code from minizip, which in turn calls zlib-functions..